test/UnitTests/GitHub.App/Models/ModelServiceTests.cs

```diff
@@ -157,20 +155,20 @@ public class TheGetAccountsMethod : TestBaseClass
             apiClient.GetOrganizations().Returns(orgs.ToObservable());
             var cache = new InMemoryBlobCache();
             var modelService = new ModelService(apiClient, cache, Substitute.For<IAvatarProvider>());
-            await modelService.InsertUser(new AccountCacheItem { Login = "snoopy" });
+            //await modelService.InsertUser(new AccountCacheItem { Login = "snoopy" });
 
             var fetched = await modelService.GetAccounts();
 
-            Assert.That(3, Is.EqualTo(fetched.Count));
-            Assert.That("snoopy", Is.EqualTo(fetched[0].Login));
-            Assert.That("github", Is.EqualTo(fetched[1].Login));
-            Assert.That("fake", Is.EqualTo(fetched[2].Login));
+            Assert.AreEqual(3, fetched.Count);
+            Assert.AreEqual("snoopy", fetched[0].Login);
+            Assert.AreEqual("github", fetched[1].Login);
+            Assert.AreEqual("fake", fetched[2].Login);
```

Comment:
The `Assert.That` constraint model is preferred for nunit 3 apparently? See https://github.com/github/VisualStudio/pull/1411#issuecomment-358045651 and the surrounding comments for context on this.

Reply:
Ah, ok. It's just that it inverts the order of the parameters, makes it even more confusing than usual. I'll fix these then.

Reply:
Yeah, I think we need to switch the order of parameters in quite a few tests if we want to be technically correct. However, it doesn't actually impact the testing and it's tedious work...